### PR TITLE
Subscribe to changes

### PR DESCRIPTION
**Pull request by szha**
## Description ##
Subscribing to changes on:
Build logic, Python frontend, Docs, Submodules, Examples, Tools, Github templates

## Comments ##
- Subscribing to changes I'd like to monitor.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -25,3 +25,27 @@ CMakeLists.txt		@cjolivier01
 Jenkinsfile         @marcoabreu
 .travis.yml         @marcoabreu
 appveyor.yml        @marcoabreu
+
+# Build logic
+Makefile          @szha
+CMakeLists.txt    @szha
+/cmake/           @szha
```
Comment (piiswrong) [inline CODEOWNERS]: these entries already exist

File: CODEOWNERS [hunk 2]
```diff
@@ -25,3 +25,27 @@ CMakeLists.txt		@cjolivier01
...
+prepare_mkl.sh    @szha
+
+# Python frontend
+/python/          @szha
```
Comment (piiswrong) [inline CODEOWNERS]: part of language bindings

File: CODEOWNERS (hunk 1 above)
Comment (szha) [reply CODEOWNERS]: Good catch. Thanks.

File: CODEOWNERS (hunk 2 above)
Comment (szha) [reply CODEOWNERS]: What do you mean?

**Pull request merged**